### PR TITLE
Forenames or forename1 now mandatory

### PR DESCRIPTION
**Pull request by timgent**

### CODE REVIEW

File: lev-api.md [hunk 1]
```diff
@@ -478,6 +481,10 @@ Get a list of births. Returns an empty list on no results found
     + id: 500152046 (required)
 
 + Request
+
+    + Parameters
+    
+        + id: 500152046 (required)
```
Comment (daniel-ac-martin) [inline lev-api.md]: Just out of interest, why was this added?

File: .dockerignore [hunk 2]
```diff
@@ -0,0 +1 @@
+node_modules
```
Comment (daniel-ac-martin) [inline .dockerignore:2]: This should probably be a copy of .gitignore shouldn't it?
Not that it's important.

**Comment by daniel-ac-martin** [comment]
LGTM.

File: lev-api.md (hunk 1 above)
Comment (timgent) [inline lev-api.md]: I think I wasn't paying attention to what I'd been messing with before :s Will remove

File: .dockerignore (hunk 2 above)
Comment (timgent) [inline .dockerignore:2]: Done
